examples/1_Santanilla_Nanomole_HTE/example_Santanilla.ipynb

```diff
@@ -0,0 +1,8234 @@
+       "<path class=\"bond-29\" d=\"M 1322.78,241.478 L 1333.21,229.459\" style=\"fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<path class=\"bond-23\" d=\"M 1307.16,238.461 L 1301.96,223.424\" style=\"fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<path class=\"bond-24\" d=\"M 1301.96,223.424 L 1312.39,211.404\" style=\"fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<path class=\"bond-24\" d=\"M 1305.69,223.497 L 1312.99,215.084\" style=\"fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<path class=\"bond-25\" d=\"M 1312.39,211.404 L 1328.01,214.422\" style=\"fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<path class=\"bond-26\" d=\"M 1328.01,214.422 L 1333.21,229.459\" style=\"fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<path class=\"bond-26\" d=\"M 1326.08,217.613 L 1329.72,228.139\" style=\"fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<text dominant-baseline=\"central\" style=\"font-size:7px;font-style:normal;font-weight:normal;fill-opacity:1;stroke:none;font-family:sans-serif;fill:#FF7F00\" text-anchor=\"middle\" x=\"1385.01\" y=\"168.112\"><tspan>P</tspan></text>\n",
+       "<text dominant-baseline=\"central\" style=\"font-size:7px;font-style:normal;font-weight:normal;fill-opacity:1;stroke:none;font-family:sans-serif;fill:#000000\" text-anchor=\"start\" x=\"1379.93\" y=\"215.843\"><tspan>Pd</tspan></text>\n",
+       "<text dominant-baseline=\"central\" style=\"font-size:7px;font-style:normal;font-weight:normal;fill-opacity:1;stroke:none;font-family:sans-serif;fill:#00CC00\" text-anchor=\"start\" x=\"1422.57\" y=\"239.737\"><tspan>Cl</tspan></text>\n",
+       "<text dominant-baseline=\"central\" style=\"font-size:7px;font-style:normal;font-weight:normal;fill-opacity:1;stroke:none;font-family:sans-serif;fill:#0000FF\" text-anchor=\"end\" x=\"1356.6\" y=\"248.826\"><tspan>H</tspan><tspan style=\"baseline-shift:sub;font-size:5.25px;\">2</tspan><tspan>N</tspan></text>\n",
+       "<path class=\"bond-0\" d=\"M 1781.41,143.941 L 1796.63,148.565\" style=\"fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<path class=\"bond-1\" d=\"M 1796.63,148.565 L 1810.48,159.608\" style=\"fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<path class=\"bo
```

Comment:
It looks like you're not doing this; please remove the comment and the commented-out code.

Reply:
All comments below this are referencing the same lines of raw notebook code that I can't really tell what they refer to. Could you direct me to the code? I cleaned up all the commented out code I could find. Meant to do this before submitting PR, though some issues Re: yield quantification were still up in the air at the time.

Reply:
Sorry, this is specifically about:

```python
# Handle any cases where 'IS' = 0 and 0 division failed
# Reset any yields greater than 105% (ORD limit)
for column in ['Product_yield', 'ArBr_yield', 'Nu_yield']:
    data[column] = data[column].fillna(0)
    #data.loc[data[column] > 105, column] = 105
```

Reply:
Done, thanks. Missed that one.